std/uri.d

```diff
@@ -38,20 +38,21 @@ private import std.traits : isSomeChar;
 import core.exception : OutOfMemoryError;
 import std.exception : assumeUnique;
 
+/** This Exception is thrown if the something goes wrong when encoding or
+decoding and uri.
```

Comment:
Grammar: `... when encoding or decoding a URI`. (Before "u" the article "a" does not become "an" in many cases.)